Sming/Components/Network/src/Network/MailMessage.h

```diff
@@ -38,6 +38,15 @@ class MailMessage
 	String subject;
 	String cc;
 
+	~MailMessage()
+	{
+		delete stream;
+		for(auto attachment : attachments) {
+			delete attachment.headers;
+			delete attachment.stream;
+		}
+	}
+
```

Comment:
Looks like `stream` should be `unique_ptr`. Also not sure it's necessary to destroy attachments, should happen automatically when vector is destroyed.

Reply:
Just done some checking and of course attachment has no destructor...